docs/pages/workflow/configuration.mdx

```diff
@@ -59,7 +59,7 @@ Library authors can extend the app config by using [Expo Config plugins](/config
 For more customization, you can use the JavaScript (**app.config.js**) or [TypeScript](#using-typescript-for-configuration-appconfigts-instead-of) (**app.config.ts**). These configs have the following properties:
 
 - Comments, variables, and single quotes.
-- Importing/requiring other JavaScript files. Using import/export syntax in external files is not supported. All imported files must be transpiled to support your current version of Node.js.
+- `import` / `export` syntax is not supported, except when using [TypeScript with ts-node](https://docs.expo.dev/guides/typescript/#appconfigjs). JS files can be imported with `require()`, but they must be transpiled to support your current version of Node.js.
```

Comment:
```suggestion
- The `import` / `export` syntax is not supported, except when using [TypeScript with `ts-node`](/guides/typescript/#appconfigjs). JS files can be imported with `require()`. However, they must be transpiled to support your current version of Node.js.
```

Reply:
Reworked this all together, hopefully it's better